woocommerce/woocommerce
# [e2e tests workflows] Run non HPOS e2e tests on push instead of PR

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





- Removed the `pull_request` trigger and added a `push` to trunk and release branches trigger for e2e tests with HPOS disabled. Hopefully this will free up some runners.
- Removed the `Evaluate e2e tests results job`, no longer needed as there is nothing required on push.
- Renamed the workflow.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Check CI. `Run tests against PR in an environment with HPOS disabled` should not run as part of this PR checks. After merging, the same checks should run.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
